UI/css/style.css

```diff
@@ -0,0 +1,638 @@
+  font-size: 1rem;
+  border: none;
+  border-radius: .2rem;
+  padding: .5rem 1.2rem;
+  margin: 1.5rem auto 0;
+  cursor: pointer;
+}
+
+.item-btn:hover {
+  background-color: rgb(17, 17, 206);
+}
+
+#order-btn {
+  padding: 1rem 5rem;
+  border: none;
+  background-color: #1d701d;
+  font-size: 1.2rem;
+  color: #fff;
+  cursor: pointer;
+  transition: 0.3s;
+}
+
+#order-btn:hover {
+  background-color: #0b490b;
+  padding: 1.2rem 5.2rem;
+}
+
+
+#orders {
+  flex-wrap: wrap;
+  display: grid;
+  grid-gap: 0.5rem;
+  grid-template-columns: repeat(auto-fit, minmax(10rem, 1fr));
+}
+
+.order-items {
+  position: relative;
+  border: .05rem solid #aaa;
+  background-color: #efefef;
+  padding: 1rem;
+  border-radius: 0.2rem;
+  min-height: 8rem;
+  box-shadow: 0 0 .3rem #666;
+  text-align: center;
+  display: flex;
+  flex-wrap: wrap;
+  text-align: left;
+}
+
+.order-items img {
+  min-width: 4rem;
+  max-width: 8rem;
+  max-height: 8rem;
+  margin: 1rem 0;
+  display: block;
+}
+
+.order-props {
+  min-height: 2rem;
+  margin: 1rem;
+  margin-right: 0;
+}
+
+#order-form input[type="number"] {
+  font-size: 1rem;
+  font-weight: 1000;
+  padding-left: 0.5rem;
+  border: .01rem solid #ccc;
+  outline: none;
+}
+
+.qty {
+  width: 3rem;
+}
+
+.qty::-webkit-inner-spin-button {
+  opacity: 1;
+}
+
+#order-form .price,
+#order-form .total{
+  width: 5rem;
+  background-color: transparent;
+  border: none !important;
+  color: #000;
+}
+
+#order-form {
+  text-align: center;
+}
+
+#order-form label {
+  text-transform: uppercase;
+  font-weight: 500;
+}
+
+#order-amount {
+  font-size: 2rem;
+  margin: 1rem;
+  padding: 1rem;
+}
+
+#order-total {
+  font-size: 2.5rem !important;
+  width: 8rem;
+  background-color: transparent;
+  border: none !important;
+  color: #000;
+}
+
+#main-footer {
+  background-color: #ccc;
+  color: #333;
+  font-weight: bold;
+  padding: 1.2rem;
+  height: 5rem;
+  text-align: center;
+  margin-top: -5rem;
+}
+
+.fa-caret-square-down {
+  position: fixed;
+  left: 1rem;
+  bottom: 5rem;
+  font-size: 3.4rem;
+  opacity: 0.7;
+  color: #333;
+}
+
+.fa-trash {
+  color: rgb(201, 14, 14);
+  font-size: 1.5rem;
+  position: absolute;
+  bottom: 1rem;
+  right: 0.7rem;
+}
+
+.fa-trash:hover,
+.accept:hover, .decline:hover,
+.unaccept:hover, .complete:hover {
+  opacity: 0.75;
+}
+
+.order-table {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  text-align: center;
+  min-height: 3rem;
+  font-weight: 500;
+  font-size: 1.1rem;
+}
+
+.order-table > div {
+  padding-top: .6rem;
+}
+
+.order-table a {
+  display: block;
+  text-decoration: none;
+  color: #000;
+}
+
+ul .order-table > div:first-child {
+  background-color: #dadada;
+}
+
+ul .order-table > div:first-child:hover {
+  background-color: #4f4f4f;
+}
+
+ul .order-table > div:first-child:hover a {
+  color: #fff;
+}
+
+.order-table.head {
+  text-transform: uppercase;
+  background-color: #aaa;
+  padding: 0 .3rem;
+}
+
+#new-orders, #accepted-orders,
```

Comment:
I am not exactly sure why this aren't all in the same class?

Reply:
That's one of the things I am planning of doing when refactoring the code

Reply:
Please do that in this pull request before merging.